arangod/Aql/Ast.cpp

```diff
@@ -2221,8 +2222,10 @@ TopLevelAttributes Ast::getReferencedAttributes(AstNode const* node, bool& isSaf
 
       if (it == result.end()) {
         // insert variable and attributeName
-        result.emplace(variable, std::unordered_set<std::string>(
-                                     {std::string(attributeName, nameLength)}));
+        result.emplace(variable,
```

Comment:
This change is just unnecessary, as here it is guaranteed that the emplace will work.
Line 2222/2223 makes sure there is no previous entry available in `result` for the key.

Reply:
not the change - the other find:)